README.md

```diff
@@ -1,21 +1,58 @@
-# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 Olympick app 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
 
-Currently there is no way to create a viewing schedule for the Olympics that only includes specific sports.
+# 🇨🇦 🇬🇹 🇰🇷 🇱🇧 🇯🇵 Olympick 🏴󠁧󠁢󠁷󠁬󠁳󠁿 🇵🇰 🇮🇷 🇬🇧 🇨🇴 
 
-This app allow a user to : <br/>
- <li>input their favourite Olympic sport's name and get all the schedules of it.</li><br/>
- <li>input their favourite Olympic sport's name  and get the schedules of specific event of it.</li><br/>
+Currently there is no way to create a viewing schedule for the Olympics that only include specific sports. 
 
```

Comment:
This is the problem we are solving, we should explain more about what the problem is.
we can add: Currently, there is no way to create a viewing schedule for the Olympics that only include specific events of the specific sports.
we are trying to solve this problem to help people to make their own personalised schedule for watching their favourite events of their favourite sports in the Uk time.
and after that they can remove the events they have watched or they do not watch it anymore from their schedules

Reply:
I think this is already in the 'this app allows a user to' section, but I'll add it in anyway!